### PR TITLE
[relnotes] Mention cling opt; cmake 3.16; C++14; rm memstat, alien depr. [skip-CI]

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

**Review by jalopezg-r00t — APPROVED** [review]
LGTM, but see comments.  :-)

File: README/ReleaseNotes/v626/index.md [hunk 1]
```diff
@@ -61,12 +61,17 @@ The following people have contributed to this new version:
...
-- RooFit container classes marked as deprecated with this release: `RooHashTable`, `RooNameSet`, `RooSetPair`, and `RooList`. These classes are still available in this release, but will be removed in the next one. Please migrate to STL container classes, such as `std::unordered_map`, `std::set`, and `std::vector`.
-- The `RooFit::FitOptions(const char*)` command to steer [RooAbsPdf::fitTo()](https://root.cern.ch/doc/v628/classRooAbsPdf.html) with an option string in now deprecated and will be removed in ROOT v6.28. Please migrate to the RooCmdArg-based fit configuration. The former character flags map to RooFit command arguments as follows:
+The "Virtual MonteCarlo" facility VMC (`montecarlo/vmc`) has been removed from ROOT. The development of this package has moved to a [separate project](https://github.com/vmc-project/). ROOT's copy of VMC was deprecated since v6.18.
+The previously deprecated packages memstat, memcheck have been removed. Please use for instance `valgrind` or memory sanitizers instead.
+ROOT's "alien" package has been deprecated and will be removed in 6.18. Please contact ALICE software support if you still rely on it.
```
Comment (bellenot) [inline README/ReleaseNotes/v626/index.md:66]: > "alien" package has been deprecated and will be removed in 6.18 
6.18? Are you sure? I suppose you meant 6.28?

**Review by bellenot — CHANGES_REQUESTED** [review]

File: README/ReleaseNotes/v626/index.md [hunk 2]
```diff
@@ -91,12 +95,31 @@ int f() { return; }
...
 
+ROOT now interprets code with optimization (`-O1`) by default, with
+proper inlining optimization and alike. This accelerates especially "modern"
+code (C++ stdlib, RDataFrame, etc) significantly. According to our measurements
+the increased time for just-in-time compiling code is reasonable given the
+runtime speed-up. Optimization can be switched with `.O 0`, `.O 1`, etc; the
+current optimization level is shown by `.O`.
+The CPP macro `NDEBUG` is now set unconditionally for interpreted code.
+Note that symbols that have been emitted with a given optimization level will not get
+
```
Comment (jalopezg-r00t) [inline README/ReleaseNotes/v626/index.md:113]: Stale newline here.  :-)
```suggestion
```

**Review by bellenot — APPROVED** [review]
LGTM

**Pull request merged**